packages/webapp/src/components/ContentFilterHeading/calcPrettyName.js

```diff
@@ -1,10 +1,10 @@
 import { titleCase } from 'change-case';
 
-const calcPrettyName = government => {
-  if(government === 'south-africa') {
+const calcPrettyName = title => {
+  if (title === 'south-africa') {
```

Comment:
This doesn't make sense any more and is probably something the view should pass in rather than have such domain-specific knowledge in the component